httpclient/src/main/java/feign/httpclient/ApacheHttpClient.java

```diff
@@ -186,6 +187,7 @@ Response toFeignResponse(HttpResponse httpResponse, Request request) throws IOEx
     }
 
     return Response.builder()
+        .protocolVersion(HTTP_PROTOCOL_VERSION)
```

Comment:
remove, let Response default value deal with this one.

Reply:
done